### ISSUE TITLE
softmax dim of spatial_ocr_block

### ISSUE THREAD

**Issue by GriffinLiang** [openseg-group/openseg.pytorch#33]
https://github.com/openseg-group/openseg.pytorch/blob/0acca42d002a89f702e8d70baf6637cfcddd8a32/lib/models/modules/spatial_ocr_block.py#L64

probs = F.softmax(self.scale * probs, dim=2)# batch x k x hw

Quick question: dim should be 1 or 2? In my opinion, k represents the number of object classes. Maybe I misunderstood some detailed parts of the proposed method.

**Comment by GriffinLiang** [comment]
it is a spatial softmax.

**Comment by PkuRainBow** [comment]
Yes, you are right
